### ISSUE TITLE
Mark - Community Amsterdam - A11y getest

### ISSUE THREAD

**Issue by rianrietveld** [nl-design-system/candidate#202]
## Review toegankelijkheid voor de Mark component in Community bij Amsterdam

- Te reviewen: [Mark in StoryBook Amsterdam](https://designsystem.amsterdam/?path=/docs/components-text-mark--docs)
- Acceptatiecriteria: [NL Design System Acceptatiecriteria voor Mark](https://nldesignsystem.nl/mark)

### Hoe te testen

Ga naar de te reviewen component in StoryBook
- Bij de component, kies in de zijbalk de view Default
- Klik op het open nieuwe tab icon rechtsboven in de taakbalk.
- Je ziet nu alleen de component met de bijbehorende CSS.

## Review toegankelijkheid door Rian op 25 november 2024

Gereviewd Mark component in StoryBook Amsterdam
- [Default](https://designsystem.amsterdam/iframe.html?globals=&args=&id=components-text-mark--default&viewMode=story)

HTML basisvoorbeeld:
```
<p class="ams-paragraph">Daarom organiseren we in 2024 het burgerberaad schone stad, waarin 150 Amsterdammers in gesprek gaan over hoe we de stad beter schoonhouden. <mark class="ams-mark">Wat vinden Amsterdammers belangrijk?</mark> Welke oplossingen zien zij? Hier zijn we benieuwd naar. Want elke Amsterdammer heeft afval en moet het kwijt. Wij kunnen als gemeente veel van deze afvalexperts leren.</p>
```

#### Acceptatiecriteria

Samenvatting: de Mark component in Community bij Amsterdan voldoet niet aan de acceptatiecriteria voor toegankelijkheid van het NL Design System.
Fail voor 
- WCAG [1.4.11 Contrast van niet-tekstuele content](https://nldesignsystem.nl/wcag/1.4.11).
- WCAG [1.4.1 Gebruik van kleur](https://nldesignsystem.nl/wcag/1.4.1)

### Toegankelijkheid algemeen

- [x] Tekst in een andere taal heeft het juiste lang-attribuut
WCAG [3.1.2 Taal van onderdelen](https://nldesignsystem.nl/wcag/3.1.2)
Oordeel: Niet van toepassing, contextafhankelijk.

### Toegankelijkheid visueel ontwerp

- [ ] De gemarkeerde tekst is niet alleen herkenbaar aan de achtergrondkleur
WCAG [1.4.1 Gebruik van kleur](https://nldesignsystem.nl/wcag/1.4.1)
Oordeel: Voldoet niet

- [x] De gemarkeerde tekst heeft voldoende contrast met de achtergrondkleur
WCAG [1.4.3 Contrast (minimum)](https://nldesignsystem.nl/wcag/1.4.3)
Contrast: #00000 tov #FFE600 = 16.57
Oordeel: Voldoet.

- [ ] Wanneer de achtergrondkleur van de Mark component anders is dan de default browserkleur, dan heeft deze een kleurcontrast van minimaal 3:1 met de achtergrondkleur van de pagina
WCAG [1.4.11 Contrast van niet-tekstuele content](https://nldesignsystem.nl/wcag/1.4.11)
Contrast: #FFFFFF tov #FFE600 = 1.26
Oordeel:  Voldoet niet.

### Toegankelijkheid zoom en herschalen

- [x] Als je inzoomt naar 200% blijft alle tekst leesbaar
WCAG [1.4.4 Herschalen van tekst](https://nldesignsystem.nl/wcag/1.4.4)
Oordeel: Voldoet.

- [x] Als je de tekstafstand vergroot blijft de tekst in zijn geheel zichtbaar
WCAG [1.4.12 Tekstafstand](https://nldesignsystem.nl/wcag/1.4.12)
Oordeel: Voldoet

### Toegankelijkheid screenreader
- [x] Gebruik het juiste HTML-element voor de Mark component
WCAG [1.3.1 Info en relaties](https://nldesignsystem.nl/wcag/1.3.1)
Oordeel: Voldoet.

## Opmerkingen
- Voeg een extra visueel aspect toe om de gemarkeerde tekst heen, bijvoorbeeld een outline, naast alleen een achtegrondkleur.
- De screenreader support van `<mark>` is wisselend. Adria Roselli geeft wat oplossingen in: [Tweaking Text Level Styles](https://adrianroselli.com/2017/12/tweaking-text-level-styles.html).
